lib/common-types.ts

```diff
@@ -566,6 +586,27 @@ export type MRActions = {
       return: AssetKeyProps
     }
   }
+  BulkAction: {
+    get: {
+      params: GetBulkActionParams
+      return: BulkActionProps
+    }
+    publish: {
+      params: GetSpaceEnvironmentParams
+      payload: any
```

Comment:
Is the `any` intentional on these? Or is that tightened up somewhere else?

Reply:
🤦🏽  forgot to add them there, fixed 👍🏽